test/pages/CommodityCodePageSpec.scala

```diff
@@ -28,4 +31,19 @@ class CommodityCodePageSpec extends PageBehaviours {
 
     beRemovable[String](CommodityCodePage)
   }
+
+  "cleanup" - {
+    "should reset CommodityCodePage" - {
+      "when HasCommodityCodePage is changed to No" in {
+        val emptyUserAnswers = UserAnswers("id", DraftId(1))
+
+        val ua = emptyUserAnswers
+          .set(CommodityCodePage, "123456")
+          .get
+
+        HasCommodityCodePage.cleanup(Some(false), ua) mustBe
+          Success(emptyUserAnswers)
+      }
+    }
+  }
```

Comment:
I feel like this should be in `HasCommodityCodePageSpec`

Reply:
Agreed